### PR TITLE
Google oauth button

### PR DESCRIPTION
**Pull request by lindsayrmaher**
#### What's this PR do?

This PR adds a google auth button to `register` and `login`! It's styled according to Google brand guidelines and tries to match the mocks as best as possible. 

@aaronschachter I think I need to add the client from Google in order to test that the link is working as expected. Can you remind me where to find that in the Google Cloud Platform and where I need to add it in my local NS?

#### How should this be reviewed?

👀 See attached screenshots for both pages & code updates

[Large - Login](https://user-images.githubusercontent.com/15236023/67218424-5a0ecd00-f3f4-11e9-909a-0cd2bf8129cb.png)
[Large - Register](https://user-images.githubusercontent.com/15236023/67218425-5a0ecd00-f3f4-11e9-995c-b4e6aaf81069.png)
[Medium - Login](https://user-images.githubusercontent.com/15236023/67218427-5a0ecd00-f3f4-11e9-81b8-9d13cc93350c.png)
[Medium - Register](https://user-images.githubusercontent.com/15236023/67218428-5a0ecd00-f3f4-11e9-95ed-529faab157db.png)
[Small - Login](https://user-images.githubusercontent.com/15236023/67218434-5da25400-f3f4-11e9-9a26-b629a876f9d7.png)
[Small - Register](https://user-images.githubusercontent.com/15236023/67218435-5da25400-f3f4-11e9-9017-044a2a0efb33.png)

#### Relevant Tickets

[168904387](https://www.pivotaltracker.com/story/show/168904387)

#### Checklist
- [ ] Adds sm, md, and lg screenshots

### CODE REVIEW

File: resources/assets/scss/_components/_facebook-login.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@ $facebook-blue: #4267b2;
   background: $facebook-blue;
   text-transform: none;
   margin: $base-spacing / 2 0;
-
+  font-size: 21px;
```
Comment (katiecrane) [inline resources/assets/scss/_components/_facebook-login.scss]: Why did Facebook styles change?

File: resources/views/auth/google.blade.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    @parent
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+@endsection
+
+    <a href="{{ url('google/continue') }}" class="button google-login md:w-3/4" style="font-family:'Roboto'" >
+        @include('icons.google-icon')
+        Continue with Google
+    </a>
+
+{{-- <a href="{{ url('google/continue') }}"><img src="/images/btn_google_signin_light_pressed_web@2x.png" /></a> --}}
```
Comment (katiecrane) [inline resources/views/auth/google.blade.php]: Not sure if you still need this for testing, but we'll want to remove it before merging ✍️

**Comment by aaronschachter** [comment]
The Google setup is documented over in https://github.com/DoSomething/northstar/blob/master/documentation/social-login.md#google -- but I don't think you need to verify the Google authentication works as expected -- the Northstar URL `/google/continue` is correct in this PR, and that work has been done in #927.

We'd need to add you (and all other devs, potentially) into the Google Cloud project to make changes in order to add your ngrok URL into the OAuth configuration, and I think for the most part you don't need to worry about that locally.

File: resources/assets/scss/_components/_facebook-login.scss (hunk 1 above)
Comment (lindsayrmaher) [reply resources/assets/scss/_components/_facebook-login.scss]: 🤔 i don't feel like there was a specific reason other than trying to get the sizing of the buttons aligned. I reverted it back and adjusted the padding on the google button instead

File: resources/assets/scss/_components/_facebook-login.scss [hunk 3]
```diff
@@ -4,7 +4,6 @@ $facebook-blue: #4267b2;
   background: $facebook-blue;
   text-transform: none;
   margin: $base-spacing / 2 0;
-
```
Comment (katiecrane) [inline resources/assets/scss/_components/_facebook-login.scss]: sorry but now there's a missing newline here 😬

File: resources/assets/scss/_components/_google-login.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      height: 32px;
+      width: 32px;
+  }
+  
+  &:hover, &:active {
+    background: $google-blue;
+  }
+
+ 
+}
```
Comment (aaronschachter) [inline resources/assets/scss/_components/_google-login.scss]: Nit: this file is missing a line break at the end, and we could remove the two extra line breaks above this `{`

**Pull request merged**